templates/vue/src/components/Lightbox/media/TapestryMedia.vue

```diff
@@ -163,7 +163,9 @@ export default {
 
 <style lang="scss" scoped>
 .media-wrapper {
+  color: #111;
```

Comment:
Again, not sure if we need this on a child element - can we keep it on the parent elements instead? Is there an overwrite somewhere that should be removed instead?

Reply:
Yeah, the rule turns out to be unnecessary, since the wrapper only houses the real element that display text below